### DISCUSSION TITLE
third-party plugins available for users

### DISCUSSION THREAD

**Discussion by BeckeBauer** [kimai/kimai#2970] [General]
When I install a third party plugin, e.g. to share timesheets with customers (https://www.kimai.org/store/vettersolutions-shared-project-timesheets-bundle.html), the plugin is only availble to the system admins. Is there a way to make it available for use to a certain group (e.g. group users) like it can be done with the Custom CSS plugin (https://www.kimai.org/store/keleo-css-custom-bundle.html) in the roles settings?

**Comment by kevinpapst** [comment]
Somehow the wrong place to ask, I would say: better ask in the bundle repository to make sure that @dexterity42 will notice your feature request.

The decision was made by the developer to hard-wire it to the Super-Admin, see eg. these lines:
https://github.com/dexterity42/SharedProjectTimesheetsBundle/blob/master/EventSubscriber/MenuSubscriber.php#L49
https://github.com/dexterity42/SharedProjectTimesheetsBundle/blob/master/Controller/ManageController.php#L28

While the CustomCSSBundle ships its own permissions:
https://github.com/Keleo/CustomCSSBundle/blob/master/DependencyInjection/CustomCSSExtension.php#L39
And then make use of them:
https://github.com/Keleo/CustomCSSBundle/blob/master/Controller/CustomCssController.php#L23
https://github.com/Keleo/CustomCSSBundle/blob/master/EventSubscriber/MenuSubscriber.php#L56

With these links, it should be easy for @dexterity42 to introduce a set of permissions.

**Reply by BeckeBauer** [reply]
Thanks for the clarification that the main program is not a 'show stopper' as regards plugins and altering permissions. I will pass on your information to dexterity42...
